x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportPutTrainedModelAction.java

```diff
@@ -341,6 +343,35 @@ protected void masterOperation(
         );
     }
 
+    void verifyMlNodesAndModelArchitectures(TrainedModelConfig configToReturn, Client client, ThreadPool threadPool) {
```

Comment:
no, because I am spying on them. Or, at least, I would have to refactor my tests to support this being a static function. I am not sure why it would be beneficial to be static though, as it should only be called from within this class (or the tests)